### DISCUSSION TITLE
[Lab 3] Vivado simulation ERROR: file could not be opened

### DISCUSSION THREAD

**Discussion by asdshawn** [bol-edu/HLS-SOC-Discussions#117] [Q&A] [answered]
Hi,
The error is, as you can see below: 
![ERRO1](https://github.com/bol-edu/HLS-SOC-Discussions/assets/31488586/1650bf1e-e3e4-46d8-a986-6b7d8dc4b80a)
and this is part of the tb code:
![ERRO2](https://github.com/bol-edu/HLS-SOC-Discussions/assets/31488586/556f6d78-f0cf-40a5-90b4-cc6462bc51a0)

Is there anyone know what could the problem be?
Thx.

**Comment by huankai0731** [comment]
Hi, you can check whether they were in the same folder with your rtl and testbench code.

**Reply by asdshawn** [reply]
Hi, whether I put them in /rtl or /tb, the same result occurs.
but still appreciate your advice, thx.

**Reply by huankai0731** [reply, marked as answer]
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/94675784/dac4d1f0-39af-4c98-bc46-2d62756de2f5)
Hi, maybe you can put them into your vivado project as design source.
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/94675784/09bff5d8-6657-401d-99db-b8843a01e2e8)

**Reply by asdshawn** [reply]
Problem solved! thank you very much :)
